core/client/app/routes/application.js

```diff
@@ -29,6 +28,20 @@ export default Ember.Route.extend(ApplicationRouteMixin, ShortcutsRoute, {
         return tokens.join(' - ') + ' - ' + this.get('config.blogTitle');
     },
 
+    sessionAuthenticated: function () {
+        let appController = this.controllerFor('application'),
```

Comment:
I think we want to default to `const` rather than `let` for things like this where we don't expect/want our references to be changed out from underneath us.